ansible_base/resource_registry/apps.py

```diff
@@ -52,25 +54,34 @@ def initialize_resources(sender, **kwargs):
         # Create resources
         for r_type in ResourceType.objects.filter(migrated=False):
             resource_model = apps.get_model(r_type.content_type.app_label, r_type.content_type.model)
+            resource_config = registry.get_config_for_model(resource_model)
 
             logger.info(f"adding unmigrated resources for {r_type.name}")
 
             data = []
             for obj in resource_model.objects.all():
-                data.append(Resource.init_from_object(obj, resource_type=r_type))
+                data.append(init_resource_from_object(obj, resource_model=Resource, resource_type=r_type, resource_config=resource_config))
 
             Resource.objects.bulk_create(data, ignore_conflicts=True)
             r_type.migrated = True
             r_type.save()
 
 
+def connect_signals(*args, **kwargs):
+    from django.db.models.signals import post_delete, post_save
+
+    from ansible_base.resource_registry.signals import handlers
+
+    post_save.connect(handlers.update_resource)
```

Comment:
What model does this signal connect to? Expectation is that the post_save signal fires whenever a registered resource is saved.

Reply:
Oh dear. Can we please have this do:

```python
for cls in get_resource_models():
    post_save.connect(handlers.update_resource, sender=cls)
```

Reply:
beats me. The sender needs to be the model, so i figured that leaving this null would set the sender to the model (which seems to be the case). I can try setting it to self and see what happens